src/trading/CancelOrder.sol

```diff
@@ -17,12 +18,12 @@ import 'ROOT/trading/IOrders.sol';
  * @title CancelOrder
  * @dev This allows you to cancel orders on the book.
  */
-contract CancelOrder is Controlled, ReentrancyGuard, ICancelOrder {
+contract CancelOrder is Controlled, CashWrapper, ReentrancyGuard, ICancelOrder {
```

Comment:
`CashWrapper is Controlled`, which makes the `Controlled` reference here redundant.  I believe diamond inheritance is supported, but there is something that needs to be considered about the order you inherit from.  It may be easier to just remove `Controlled` here.